src/main/java/de/wejago/mqtt2influx/service/JsonSubscriber.java

```diff
@@ -35,8 +34,10 @@ public void messageArrived(String topic, MqttMessage mqttMessage) {
             if (StringUtils.isNotBlank(device.getOnlyMatch()) && receivedMessage.contains(device.getOnlyMatch())) {
                 JsonNode messageNode = objectMapper.readTree(receivedMessage);
                 Map<String, Object> deviceToPointProperties = readDeviceValuesFromMqttMessage(messageNode);
-                Point point = generateMeasurementPoint(deviceToPointProperties, device);
-                influxDbRepository.writePoint(point);
+                MqttDataPoint mqttDataPoint = generateMqttDataPoint(deviceToPointProperties, device);
+                System.out.println("mqttDataPoint size = " + mqttDataPoint.getFields().size());
```

Comment:
Output not needed, System.out should not be used in normal services! Better just use the logger.